gnd/src/main/java/com/google/android/gnd/persistence/local/room/entity/OptionEntity.java

```diff
@@ -35,7 +35,7 @@
             childColumns = "field_id", // NOPMD
             onDelete = ForeignKey.CASCADE),
     indices = {@Index("field_id")}, // NOPMD
-    primaryKeys = {"code", "field_id"}) // NOPMD
+    primaryKeys = {"id"}) // NOPMD
```

Comment:
Could you also please remove the `NOPMD` from all these 3 places and 1 more in the same file?

It was added because of 4 occurrences of string literal "field_id". Now there are only 3
https://pmd.github.io/latest/pmd_rules_java_errorprone.html#avoidduplicateliterals

Also, we should think about whether to disable this check as it is a bit annoying and less useful.

Reply:
5b11ee0 thanks!